spec.html

```diff
@@ -7243,6 +7242,289 @@ <h1>
   <h1>Syntax-Directed Operations</h1>
   <p>In addition to those defined in this section, specialized syntax-directed operations are defined throughout this specification.</p>
 
+  <emu-clause id="sec-evaluation" type="sdo">
+    <h1>Runtime Semantics: Evaluation ( ): a Completion Record</h1>
+    <dl class="header">
+    </dl>
+    <emu-note>
+      The definitions for this operation are distributed over the "ECMAScript Language" sections of this specification. Each definition appears after the defining occurrence of the relevant productions.
+    </emu-note>
+    <ul>
```

Comment:
We should put a short introduction like

> Evaluation is defined across the following sections:

Reply:
Done. I changed "Evaluation" to "The Evaluation operation" just so that it's clear we're talking about the operation with that name, and not the general idea of evaluation.

Also, this was mostly what the Note was saying, so I dropped that.